site/layouts/partials/home/masthead.html

```diff
@@ -2,7 +2,7 @@
   <div class="container-xxl bd-gutter">
     <div class="col-md-8 mx-auto text-center">
       <a class="d-flex flex-column flex-lg-row justify-content-center align-items-center mb-4 text-dark lh-sm text-decoration-none" href="https://blog.getbootstrap.com/2022/05/13/bootstrap-5-2-0-beta/">
-        <strong class="d-sm-inline-block p-2 me-2 mb-2 mb-lg-0 rounded-3 masthead-notice">New in v5.2</strong>
+        <strong class="d-sm-inline-block p-2 me-2 mb-2 mb-lg-0 rounded-3 masthead-notice">New in v{{ .Site.Params.docs_version }}</strong>
```

Comment:
I am not sure of this change. Seems it has to be fixed in 5.2

Reply:
I think I get it, I'll remove it thanks :)